stdlib/ballerina-builtin/src/main/java/org/ballerinalang/nativeimpl/io/Read.java

```diff
@@ -69,19 +103,27 @@
     public BValue[] execute(Context context) {
         BStruct channel;
         int numberOfBytes;
-        BBlob readByteBlob;
-        BInteger numberOfReadBytes;
+        int offset;
```

Comment:
Shall we define above 3 variables (channel, numberOfBytes, offset) within the try block it self. That will reduce them to a single line and more readable

Reply:
In terms of readability, having variables defined outside the try block would help understand it's usage, and the types of memory allocated for the whole process. And some of the variables are being used by catch blocks etc which will not allow to move them under the try scope